plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -596,6 +677,11 @@ GSTexture* GSDevice11::CreateSurface(int type, int w, int h, int format)
 	return t;
 }
 
+GSTexture* GSDevice11::FetchSurface(int type, int w, int h, int format)
+{
+	return __super::FetchSurface(type, w, h, format);
```

Comment:
Maybe need to check if format is not 0

Reply:
Well it should be fine as it is the same as old code.

Note: it would be nice to have a higher enum format. This way we could use the same for all API